operators/test/e2e/es/podtemplate_test.go

```diff
@@ -61,7 +61,8 @@ func TestCustomConfiguration(t *testing.T) {
 					},
 				},
 			},
-		})
+		}).
+		WithRestrictedSecurityContext()
```

Comment:
Would it be maybe cleaner to include:
```
SecurityContext: test.DefaultSecurityContext()
```

in the custom pod template we build (L37)?

Reply:
I hesitated to do that! It seems to me that `WithRestrictedSecurityContext` is cleaner but I've no strong opinion.

Reply:
No strong opinion :)